DynamicArray.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include "DynamicIntArray.h"
+
+int main()
+{
```

Comment:
Not all functionality is tested here. Try to test as much functionality as possible